authors/apps/profiles/migrations/0002_remove_profile_bio.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.1 on 2019-03-04 08:48
```

Comment:
@NLSanyu , I think this migrations are too many especially these ones for remove profile bio

Reply:
Okay, let me work on that